src/commands/stepfunctions/helpers.ts

```diff
@@ -12,9 +14,7 @@ export const displayChanges = (
   params: any,
   previousParams?: any
 ): void => {
-  context.stdout.write(`${'='.repeat(50)}`)
-  context.stdout.write(`\n${dryRun ? '\n[Dry Run] Planning for' : 'Will apply'} the following change:\n`)
-  context.stdout.write(`\nChanges for ${stepFunctionArn}\n`)
```

Comment:
Is this why we are changing the print?

Reply:
This section is too verbose and updating it would resolve the issue you raised in the original PR where step function ARN are showing too many times.

Reply:
Oh, perfect!